cloudslang-entities/src/main/java/io/cloudslang/lang/entities/AsyncLoopStatement.java

```diff
@@ -43,27 +41,4 @@ public String getVarName() {
         return varName;
     }
 
-    @Override
```

Comment:
Why did you move the toString(), equals() & hashcode() methods to the parent class? 
You miss the extending class's specific fields, such as `varName` in this case